packages/screens/RWA/RWAHomeScreen/RWAHomeScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from "react";
+import { View } from "react-native";
+
+import { HomeProposals } from "./components/HomeProposals/HomeProposals";
+import { useIsMobile } from "../../../hooks/useIsMobile";
+import { setIsLightTheme } from "../../../store/slices/settings";
+import { useAppDispatch } from "../../../store/store";
+import { ScreenFC } from "../../../utils/navigation";
+import { layout } from "../../../utils/style/layout";
+import {
+  EstateCardList,
+  getEstateCardList,
+} from "../components/EstateCard/EstateCardList";
+import { RWAScreenContainer } from "../components/RWAScreenContainer/RWAScreenContainer";
+import { useIsRWAListThreshold } from "../useIsRWAListThreshold";
+
+export const RWAHomeScreen: ScreenFC<"RWAHome"> = () => {
+  const dispatch = useAppDispatch();
+  const isMobile = useIsMobile();
+  const isRWAListThreshold = useIsRWAListThreshold();
```

Comment:
Nope it doesn't because this hook is useful to have a transition before mobile one, and just the `isSidebarExpanded` doesn't give me the specific size i want 👍